egs/wsj/s5/steps/tfrnnlm/lmrescore_rnnlm_lat_pruned.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Copyright 2015  Guoguo Chen
+#           2017  Hainan Xu
+# Apache 2.0
+
+# This script rescores lattices with RNNLM trained with TensorFlow.
+# It uses a pruend algorithm to speed up the runtime and improve the accuracy.
+# which is an improved version over steps/tfrnnlm/lmrescore_rnnlm_lat.sh,
+# which uses the exact same interface
+# The details of the pruning algorithm is described in
+# http://www.cs.jhu.edu/~hxu/tf.pdf
+
+# Begin configuration section.
+cmd=run.pl
+skip_scoring=false
+max_ngram_order=4 # Approximate the lattice-rescoring by limiting the max-ngram-order
+                  # if it's set, it merges histories in the lattice if they share
+                  # the same ngram history and this prevents the lattice from 
+                  # exploding exponentially. Details of the n-gram approximation
+                  # method are described in section 2.3 of the paper
+                  # http://www.cs.jhu.edu/~hxu/tf.pdf
+inv_acwt=10
```

Comment:
I'm not sure why you are having inv_acwt being specified, instead of acwt.  The reason we sometimes use inv_acwt is because it interacts better with queue.pl and the like, but in this case that issue doesn't exist, so I think it would be better to specify acwt directly.